src/app/interfaces/NFT/CreaturesNFT.ts

```diff
@@ -0,0 +1,30 @@
+export interface CreaturesPackInfo {
+  success: boolean;
+  result: Result;
+}
+
+export interface Result {
+  id: number;
+  name: string;
+  description: string;
+  initialSize: number;
+  collection: Collection;
+  price: number;
+  maxPerUser: number;
+  enabled: boolean;
+  hidden: boolean;
+  startTime: null;
+  endTime: null;
+  createdAt: string;
+  quoteCurrency: string;
+  numRemaining: number;
+}
+
+export interface Collection {
+  name: string;
+  twitterUrl: null;
```

Comment:
It's better to use string instead of null here because in future these may have a different value

Reply:
oh damn auto typing 😝